Directory.Build.targets

```
@@ -154,20 +164,16 @@ or
       <AndroidSdkArtifacts Include="$(RepoRoot)lib/gson-2.8.5.jar" />
     </ItemGroup>
 
-    <Copy SourceFiles="@(AndroidSdkArtifacts)"
-          DestinationFiles="@(AndroidSdkArtifacts->'$(SentryAndroidArtifactsDestination)%(RecursiveDir)%(Filename)%(Extension)')">
+    <Copy SourceFiles="@(AndroidSdkArtifacts)" DestinationFiles="@(AndroidSdkArtifacts->'$(SentryAndroidArtifactsDestination)%(RecursiveDir)%(Filename)%(Extension)')">
     </Copy>
 
     <Error Condition="!Exists('$(SentryAndroidArtifactsDestination)')" Text="Failed to build the Android SDK."></Error>
   </Target>
 
-  <!-- Build the Sentry Native SDK for Windows: dotnet msbuild /t:BuildNativeWindowsSDK src/Sentry.Unity -->
-  <Target Name="BuildNativeWindowsSDK"
-          Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'
+  <!-- Build the Sentry Native SDK for Windows: dotnet msbuild /t:BuildWindowsSDK src/Sentry.Unity -->
+  <Target Name="BuildWindowsSDK" Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'
           And $([MSBuild]::IsOsPlatform('Windows'))
-          And !Exists('$(SentryNativeWindowsArtifactsDestination)/sentry.dll')"
-          DependsOnTargets="BuildNativeSDK"
-          BeforeTargets="BeforeBuild">
+          And !Exists('$(SentryNativeWindowsArtifactsDestination)/sentry.dll')" DependsOnTargets="BuildNativeSDK" BeforeTargets="BeforeBuild">
```

With the change `BuildNativeWindowsSDK` -> `BuildWindowsSDK` we might want to rename also:

```suggestion
          And !Exists('$(SentryWindowsArtifactsDestination)/sentry.dll')" DependsOnTargets="BuildNativeSDK" BeforeTargets="BeforeBuild">
```

---

replaced `NativeWindows -> Windows`  in the file